.gitignore

```diff
@@ -6,11 +6,6 @@
 htpasswd
 .yarn
 
-# Nix
-gomod2nix.toml
```

Comment:
suggestion: Add the `gomod2nix.toml` file to the `.gitattributes` file to mark it as auto-generated.

Reply:
Oh, for some reason I thought that some some config (just because it was toml)...
Done in e2a4d70bbd67375ff57f21ec40412bf7d233152a